index.bs

```diff
@@ -1798,6 +1822,66 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Client Supported Capabilities (dictionary <dfn dictionary>SupportedCapabilities</dfn>) ### {#dictionary-supportedcapabilities}
+
+<xmp class="idl">
+    dictionary SupportedCapabilities {
+        boolean areU2FDevicesSupported;
+        boolean areCTAPDevicesSupported;
```

Comment:
Point of order: U2F == CTAP1, and CTAP2 is the new thing, right?

Seems like this should be `areCTAP2DevicesSupported`? Or maybe it should just be `minimumCTAPVersionSupported` and make it a float?

Reply:
Agreed about saying &ldquo;CTAP2&rdquo;.

Minimum version might be a problem for Safari: don't they _only_ support CTAP2 devices at the moment?